pontus_sensors/launch/camera.launch.py

```diff
@@ -0,0 +1,77 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.conditions import IfCondition
+from launch_ros.actions import Node
+import sys
+import os
+import yaml
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+
+    pontus_sensors_share = get_package_share_directory('pontus_sensors')
+    oakd_share = get_package_share_directory('depthai_ros_driver')
+    
+    imu_arg = DeclareLaunchArgument('imu', default_value='gx3')
+    imu_config = LaunchConfiguration('imu', default='gx3')
+
+    imu_config_str = 'gx3'
+    for arg in sys.argv:
+        if arg.startswith('imu:='):
+            imu_config_str = arg.split(':=')[1]
+
+    ld = list()
+    ld.append(imu_arg)
+
+    with open(f'{pontus_sensors_share}/config/oakd.yaml', 'r') \
+        as stream:
+        camera_data = yaml.safe_load(stream)
+
+    for i, topic in enumerate(camera_data['camera_config']['all_camera_base_topics']):
+        cam_type = camera_data['camera_config']['all_camera_types'][i]
+
+        if cam_type == 'oakd':
+            ld.append(
+                IncludeLaunchDescription(
+                    PythonLaunchDescriptionSource(os.path.join(oakd_share, 'launch/camera.launch.py')),
+                )
+            )
+            if imu_config_str == 'oak': # using oak-d imu
+                ld.append(Node(
+                    package='virtuoso_sensors',
+                    executable='gx3_republish',
+                    remappings=[
+                        ('/imu/data', '/oak/imu/data')
+                    ]
+                ))
```

Comment:
Let's leave this commented out for now with a note to add the ability to toggle OAK-D vs gx3 imu in the future